vividus-plugin-web-app/src/main/java/org/vividus/bdd/monitor/ScreenshotOnFailureMonitor.java

```diff
@@ -123,8 +126,10 @@ private boolean isStoryHasNoScreenshotOnFailureMeta()
 
     private boolean isScenarioHasNoScreenshotsOnFailureMeta()
     {
-        RunningStory runningStory = bddRunContext.getRunningStory();
-        return runningStory.getRunningScenario().getScenario().getMeta()
-                .hasProperty(NO_SCREENSHOT_ON_FAILURE_META_NAME);
+        return Optional.ofNullable(bddRunContext.getRunningStory())
+                       .map(RunningStory::getRunningScenario)
+                       .map(RunningScenario::getScenario)
```

Comment:
similar here: if `runningScenario` is not `null`, will `scenario` ever be `null`?